StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/Models/FinancialConnectionsGenericInfoScreen.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  FinancialConnectionsGenericInfoScreen.swift
+//  StripeFinancialConnections
+//
+//  Created by Krisjanis Gaidis on 6/20/24.
+//
+
+import Foundation
+
+struct FinancialConnectionsGenericInfoScreen: Decodable {
```

Comment:
Yea I named it for consistency with backend.

I added `FinancialConnections` prefix purely because of existing precedent/consistency. I believe it started because the public model `FinancialConnectionsSession` that's returned from the sheet has `FinancialConnections`. For this particular PR I think its OK because of current precedent, _but_ we could revisit this decision. I am usually pro-short-prefix, _but_ `FinancialConnections` is _very_ long so I don't necessarily like it either. I'd be OK with removing if you deem it better. We'd just have to do it mindfully together for the whole codebase because there might be some gotchas.